### ISSUE TITLE
Skip RDF parsing of json-ld files, parse them as JSON directly

### ISSUE THREAD

**Issue by SimonVanneste** [openplannerteam/planner.js#102]
Right now (using LDFetch):

1. JSON-LD file parsed as JSON
2. Converted to graph representation (?) using included context
3. Converted to triples (subject, predicate, object)
4. Converted to JS objects for our use

1-3 happens inside LDFetch. We should just skip steps 2 and 3

**Comment by pietercolpaert** [comment]
Once https://github.com/comunica/comunica/pull/473 is merged into master of Comunica, we should change our source to Comunica. The power of Planner.js should at some point be that it can automatically read from other sources too.

Closing this in favor of #120
